src/componentIcons.ts

```diff
@@ -1,11 +1,13 @@
 import radioButton from './componentIcons/icon-radio-button.svg';
 import radioButtonLarge from './componentIcons/icon-radio-button-large.svg';
 import toggle from './componentIcons/icon-toggle.svg';
+import loadingSpinner from './componentIcons/icon-loading-spinner.svg';
```

Comment:
Should we rename to `preloader` everywhere for the name instead as it is the name in the designs?

Reply:
Good point. I changed the name in the latest commit